.github/workflows/build-push.yml

```diff
@@ -167,10 +172,12 @@ jobs:
 
       - uses: actions/setup-python@v5
         name: Install Python
+        with:
+          python-version: '3.13'
 
       - name: Install cibuildwheel
         run: |
-          python -m pip install cibuildwheel==2.16.2
+          python -m pip install cibuildwheel==2.20.0
```

Comment:
Why here it is `2.20.0` and before it is `2.22.0` ?

Reply:
A mistake

Reply:
fixed it